src/Listener.php

```diff
@@ -148,17 +176,31 @@ public function process()
                     APILogger::addInfo('HoldRequestResult', $holdRequestResult);
 
                     if ($holdRequestResult->isSuccess() === false) {
-                        // TODO: Fetch e-mail from patron record
-                        APILogger::addInfo('E-mail time', $holdRequestResult->getHoldRequest()->getDocDeliveryData()->getEmailAddress());
+                        $holdRequest = HoldRequestClient::getHoldRequestById($holdRequestResult->getHoldRequestId());
+                        APILogger::addInfo('HoldRequest', $holdRequest);
 
-                        $patron = PatronClient::getPatronById($holdRequestResult->getHoldRequest()->getPatron());
+                        $patron = PatronClient::getPatronById($holdRequest->getPatron());
 
-                        APILogger::addInfo('Patron', $patron->getId());
+                        APILogger::addInfo('Patron', $patron);
                         APILogger::addInfo('E-mail', $patron->getEmails());
                         APILogger::addInfo('BarCodes', $patron->getBarCodes());
-                        // TODO: Send e-mail to patron notifying failure
```

Comment:
Actually I'm going to remove that logging message. I think I have enough log data to indicate something is working so e-mail and barcode are no longer needed in the logs.